.eslintrc.yml

```diff
@@ -0,0 +1,112 @@
+parser: "@babel/eslint-parser"
```

Comment:
Why babel? Why do we need even changing the default parser? 🤔

Reply:
![image](https://github.com/asyncapi/spec-json-schemas/assets/127925465/b99853aa-eb08-453f-88e2-dc73bfd7a601)

when i run `npm run lint` it gives the same error

![image](https://github.com/asyncapi/spec-json-schemas/assets/127925465/93c29007-cd58-4255-9eaa-449b9a3afe2e)

Reply:
Those errors seem genuine to me, and are errors we should fix in our code.

That screenshot about babel has nothing to do with this as far as I can see.

Reply:
@smoya i have reffered  (this)[https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token]  and have used `babel-eslint` 
I might be wrong here

Reply:
That example uses React, which iirc uses Babel.

Thats the reason they are talking about babel.
We do not need any parser different than the default one.